string name
string[] outcomes
string[] parents
string[] children
pgm_learner/ConditionalProbability[] CPT
